device/globals/game.gd

```diff
@@ -29,6 +29,12 @@ func set_mode(p_mode):
 func mouse_enter(obj):
 	var text
 	var tt = obj.get_tooltip()
+
+	# When following the mouse, prevent text from flashing for a moment in the wrong place
+	if ProjectSettings.get_setting("escoria/ui/tooltip_follows_mouse"):
+		var pos = get_viewport().get_mouse_position()
+		tooltip.set_position(pos)
```

Comment:
Does this make ` func _input(ev)` on line 336 redundant?

Reply:
No, because that's what actually tracks the mouse position and updates the tooltip location, as every movement of the mouse is an input event.

Reply:
Is this logic only triggered when the mouse enters the object then? If so, wouldn't it also be possible to remove the `vm.hover_object` condition from `func _input(ev)` to have the tooltip always track the mouse, even when it's not hovering over an object?

Reply:
I tested this, and also a bug came up. Removing the `vm.hover_object` condition leads to a crash on startup, because `tooltip` is not set. This is something like a race condition of sorts that would be more painful to hunt down, to initialize `tooltip` elsewhere.

So I think it should be as it is.

Reply:
Ok, I understand, and I think this is a good workaround for now. :slightly_smiling_face: